workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -66,6 +59,14 @@ def _prepare_input(job_context: Dict) -> Dict:
     if job_context['job'].success is not False:
         job_context["organism_name"] = job_context['group_by_keys'][0]
 
+        # TEMPORARY for iterating on compendia more quickly. Rather
+        # than downloading the data from S3 each run we're just gonna
+        # use the same directory every job.
+        job_context["old_work_dir"] = job_context["work_dir"]
+        job_context["work_dir"] = "/home/user/data_store/" + job_context["organism_name"] + "/"
```

Comment:
Should we add `/smashed/` here?

```suggestion
        job_context["work_dir"] = SMASHING_DIR + job_context["organism_name"] + "/"
```

Reply:
Sure, why not!